department-of-veterans-affairs/abd-vro
# MCP:2657: Update asthma message routing

## What was the problem?

Pair PR with #1515 
Associated tickets or Slack threads:

- [MCP-2657](https://amida.atlassian.net/browse/MCP-2657)

## How does this fix it?[^1]


## How to test this PR
- Step 1
- 
- Step 2[^secrel]


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
